contracts/assembly/src/contract.rs

```diff
@@ -773,14 +783,24 @@ pub fn calc_total_voting_power_at(deps: Deps, proposal: &Proposal) -> StdResult<
     if let Some(vxastro_token_addr) = config.vxastro_token_addr {
         // Total vxASTRO voting power
         let vxastro: VotingPowerResponse = deps.querier.query_wasm_smart(
-            vxastro_token_addr,
+            &vxastro_token_addr,
             &VotingEscrowQueryMsg::TotalVotingPowerAt {
                 time: proposal.start_time - 1,
             },
         )?;
         if !vxastro.voting_power.is_zero() {
             total = total.checked_add(vxastro.voting_power)?;
         }
+
+        let locked_xastro: BalanceResponse = deps.querier.query_wasm_smart(
+            config.xastro_token_addr,
+            &XAstroTokenQueryMsg::BalanceAt {
+                address: vxastro_token_addr.to_string(),
+                block: proposal.start_block,
+            },
+        )?;
+
+        total = total.checked_add(locked_xastro.balance)?;
```

Comment:
We already added this at line 767

Reply:
They are different. Line 767 fetches xASTRO total supply while here we fetch xASTRO balance of vxASTRO contract

Reply:
But xASTRO balance of vxASTRO contract is already included in xASTRO total supply, isn't it?

Reply:
Hm, you are right. Thank you. Need to fix it